src/dolphin/filtering.py

```diff
@@ -103,6 +106,44 @@ def filter_long_wavelength(
         return filtered_ifg
 
 
+def _fill_boundary_area(unw_ifg_interp: np.ndarray, mask: np.ndarray) -> np.ndarray:
+    """Fill the boundary area by reflecting pixel values."""
+    # Rotate the data to align the frame
+    rt_unw_ifg = ndimage.rotate(unw_ifg_interp, 8.6, reshape=False)
```

Comment:
we'll want a parameter for the rotation in degrees:
- inputs in radar coordinates will need no rotation
- ascending and descending geocoded inputs would take the opposite rotation amounts

Reply:
we'll also probably want some parameter to indicate wither there's any fill requested at all- a radar image filled all the way to the borders doesn't need this reflection